src/routes/popup/send/confirm.tsx

```diff
@@ -410,11 +444,7 @@ export function ConfirmView({
           console.log(e);
           setIsLoading(false);
           freeDecryptedWallet(keyfile);
-          setToast({
-            type: "error",
-            content: browser.i18n.getMessage("failed_tx"),
-            duration: 2000
-          });
+          showTransferError();
```

Comment:
Tested blocking the gateways, but if the gateways are not responding, we don't get to this part of the code

The `prepare` function already throws in line 209 of the code

In order to see the error message I need to block the gateways only when it's submitting the tx

Also, with gateways blocked from the start, it shows an error message and doesn't even reach the confirm screen of the flow
IMO most users with gateway issues won't see the error message